package.json

```diff
@@ -9,7 +9,7 @@
     "build": "webpack --config webpack.config.js --progress --profile --colors",
     "watch-css": "node node_modules/autoless/bin/autoless --source-map templates/less public",
     "watch-js": "webpack --config webpack.config.js --progress --profile --colors --watch",
-    "watch": "node watch.js"
+    "watch": "parallelshell \"npm start\" \"npm run watch-css\" \"npm run watch-js\""
```

Comment:
what do you thin about hving this be `npm start`, and making the current start `npm run serve`?